### DISCUSSION TITLE
ellipsis (three dots) expand and collapse the text

### DISCUSSION THREAD

**Discussion by chentao169** [quasarframework/quasar#13734] [CLI - PWA mode] [answered]
I want to make the extra text three dots (...) ellipsis and when i click the dots the text should expand and contract. But with the code used the text is only contracted but not expanding on click of dots

**Comment by pdanpdan** [comment]
What code?

**Reply by chentao169** [reply]
I found CSS class `ellipsis` https://quasar.dev/style/visibility#introduction auto truncate text, but it doesn't expand text when clicking on it...

**Comment by pdanpdan** [comment]
You can remove the class on click if you want to expand.

**Reply by chentao169** [reply]
yep, but I don't know how to apply the click action on the ellipsis...

**Reply by suleimanas** [reply]
apply the click on the whole text and toggle the ellipsis class

**Reply by chentao169** [reply]
this is kinda suboptimal. what I want is to click the ellipsis, not the whole text.

**Comment by suleimanas** [comment, marked as answer]
You cannot attach events to pseudo-content and dots here are pseudo-content. so make your own or you could find some hacky css solution.
just to get the idea: https://stackoverflow.com/a/48854854

**Reply by chentao169** [reply]
thank you!
